scripts/automation/base_python.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -x
+
+# This script installs miniconda and numpy into the dismod_at space, 
+# and upgrades pip, virtualenv, and pipenv
+
+# This should be 2 or 3.
+PYTHONVERSION=3
+
+CODE=/ihme/code/dismod_at
+WORKDIR="${CODE}/build"
+SCRIPT="Miniconda${PYTHONVERSION}-latest-Linux-x86_64.sh"
+INSTALLDIR="${CODE}/pyenv/versions"
+
+# Any existing python path can interfere with installation, as can
+# other Pythons on the path.
+unset PYTHONPATH
+export PATH=/usr/local/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin
+
+cd "${WORKDIR}"
+rm -f "${SCRIPT}"
+wget "https://repo.continuum.io/miniconda/${SCRIPT}"
+
+# The version is stored in this way in the file, so find that.
+# NAME:   Miniconda3
+# VER:    4.5.1
+CONDANAME=`awk '/NAME:/ {print tolower($3)}' ${SCRIPT}`
+CONDAVER=`awk '/VER:/ {print $3}' ${SCRIPT}`
+SUBDIR="${CONDANAME}-${CONDAVER}"
+CONDAHOME="${INSTALLDIR}/${SUBDIR}"
+
+if [ -e  "${CONDAHOME}" ]; then
+  echo Directory "${CONDAHOME}" exists so cannot install.
+  exit 37
+fi
+
+# Run the installer in batch mode.
+bash "${SCRIPT}" -b -p "${CONDAHOME}"
+
+"${CONDAHOME}/bin/conda" install -y numpy[mkl]
```

Comment:
I think we need a few more conda libraries in order to support get_draws and friends. Please add blosc, redis, hdf5, and mysql-connector-c.